advanced/irods_rule_engine_plugin_storage_balancing/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
-find_package(IRODS 4.3.0 EXACT REQUIRED CONFIG)
+find_package(IRODS 4.3.2 EXACT REQUIRED CONFIG)
```

Comment:
Alternatively, we can implement something like what we do with our newer versions so we don't have to keep touching this... https://github.com/irods/irods_rule_engine_plugin_audit_amqp/blob/2310a021680aa4142c5bc203e33a0dcce85a4ec6/CMakeLists.txt#L7-L10

Then again, we should probably just revamp this whole project eventually.

Reply:
yeah, either is fine for now - this is once per year at most...

Reply:
Okay. Let's just leave it as is for now, then.

Reply:
Opened #37. We can get that in post UGM.